code/modules/jobs/job_types/chaplain/chaplain_costumes.dm

```diff
@@ -193,8 +193,17 @@
 	flags_inv = NONE
 	icon_state = "cage"
 	inhand_icon_state = null
+	armor_type = /datum/armor/helmet_cage
 	worn_y_offset = 7
 
+/datum/armor/helmet_cage
```

Comment:
Any reason this has its own new armour datum instead of just using the existing chaplain helmet armour values?  
It's hard for me to understand why the cage helmet would be _better_ armour than the actual medieval armour helmet.